dfsclient/src/main/java/org/hpccsystems/dfs/client/HpccRemoteFileReader.java

```diff
@@ -385,8 +414,14 @@ public T next()
      */
     public void close() throws Exception
     {
+        if (isClosed)
+        {
```

Comment:
Is this an uncommon enough condition worth logging?

Reply:
Yeah, it shouldn't be a common occurrence, and while not problematic it would likely indicate a bug in the calling code. I will add a log entry.